### ISSUE TITLE
Keyboard shortcut & context menu item to toggle fullscreen mode

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/Web-Map-Custom-Element#201]
As suggested in https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/18#issuecomment-588576345, it'd be great to have a context menu item to toggle fullscreen mode.

IMO it is a highly valuable feature. It should be available even if the author chooses to disable the control in the UI using `controlslist="nofullscreen"` (disabling a control does not disable its corresponding feature, per the spec).

I believe the conventional keyboard shortcut for fullscreen is <kbd>F</kbd>, which would clash with the "Forward (<kbd>F</kbd>)" menu item. 🤔 In the gaming world it is not uncommon to navigate (e.g. move a character) using the [WASD keys](https://en.wikipedia.org/wiki/Arrow_keys#WASD_keys), so maybe that's something the Back/Forward/Reload items could use instead? 🤷‍♂️

**Comment by prushforth** [comment]
Resolved by https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/765
